Assets/Scripts/PlayerController.cs

```diff
@@ -7,7 +7,27 @@
 public class PlayerController : MonoBehaviour {
 
     [SerializeField]
-    private float playerAcceleration, maxSpeed, jumpForceForward, jumpForceUp, bulletSpeed, recoilSpeed, overheatAdd, overheatSubtract;
+    [Tooltip("Max Speed that the Player can Move")]
+    private float maxSpeed;
+    [SerializeField]
+    [Tooltip("How far Forward the Player Jumps on Jump")]
+    private float jumpForceForward;
+    [SerializeField]
+    [Tooltip("How far Up the Player Jumps on Jump")]
+    private float jumpForceUp;
+    [SerializeField]
+    [Tooltip("How Fast the Bullets Move")]
+    private float bulletSpeed;
+    [SerializeField]
+    [Tooltip("How much Force is applied to the Player when they Shoot")]
```

Comment:
So should this number be a negative? At the minimum describe if this should be pos or neg, or you could try using the range attribute to enforce it. https://docs.unity3d.com/ScriptReference/RangeAttribute.html